metricflow/api/simple_api.py

```diff
@@ -0,0 +1,259 @@
+from typing import Dict, List, Optional
+
+from metricflow.configuration.config_handler import ConfigHandler
+from metricflow.configuration.constants import CONFIG_DWH_SCHEMA
+from metricflow.dataflow.sql_table import SqlTable
+from metricflow.engine.metricflow_engine import (
+    MetricFlowEngine,
+    MetricFlowExplainResult,
+    MetricFlowQueryRequest,
+    MetricFlowQueryResult,
```

Comment:
Not now, but eventually these should probably be tidied up and moved in the api directory.